bridge/client/app/_utils/form.utils.ts

```diff
@@ -1,6 +1,8 @@
 import { AbstractControl, ValidatorFn } from '@angular/forms';
 
 export class FormUtils {
+  public static URL_PATTERN = /^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#[\]@!\$&'\(\)\*\+,;=.]+$/;
```

Comment:
I would keep the escape consistent:
```suggestion
  public static URL_PATTERN = /^(?:http(s)?:\/\/)?[\w\.\-]+(?:\.[\w\.\-]+)+[\w\-\._~:\/?#[\]@!\$&'\(\)\*\+,;=\.]+$/;
```

I think this would allow to match `https://mywebsite..com-`
```suggestion
  public static URL_PATTERN = /^(?:http(s)?:\/\/)?[\w\.-]+(?:[\w\.-]+)+[\w\._~:/?#[\]@!\$&'\(\)\*\+,;=]+$/;
```

Reply:
I would suggest to follow the RegEx: `^http(s)?:\/\/([\w-]+.)+[\w-]+(\/[\w\- .\/?%&=])?$`
source: https://stackoverflow.com/a/5717338

Reply:
This one is still wrong. Something like "mywebsite.com", "http://mywebsite.com/" or "http://mywebsite.com/?q=asdf" is not possible